### PR TITLE
Fix incorrect syntax

### PR DESCRIPTION
**Pull request by simsieg**

### CODE REVIEW

**Comment by icebob** [comment]
The syntax is correct and works.

File: source/docs/0.13/moleculer-web.md [hunk 1]
```diff
@@ -150,7 +150,7 @@ broker.createService({
     settings: {
         routes: [{
             aliases: {
-                "POST upload"(req, res) {
+                "POST upload": (req, res) {
```
Comment (icebob) [inline source/docs/0.13/moleculer-web.md]: This syntax is not correct, but original is. Please leave the original

File: source/docs/0.13/moleculer-web.md [hunk 2]
```diff
@@ -778,7 +778,7 @@ settings: {
                 "POST divide": "math.div",
                 "GET greeter/:name": "test.greeter",
                 "GET /": "test.hello",
-                "POST upload"(req, res) {
+                "POST upload": (req, res) {
```
Comment (icebob) [inline source/docs/0.13/moleculer-web.md]: This syntax is not correct, but original is. Please leave the original

Comment (simsieg) [reply source/docs/0.13/moleculer-web.md]: True, my fault :)

**Comment by icebob** [comment]
Thanks.

**Pull request merged**